test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -203,30 +165,15 @@ public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallb
         }
 
         [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        public Task Localization_ResourcesAtRootFolder_ReturnLocalizedValue_Clr()
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
+        public Task Localization_ResourcesAtRootFolder_ReturnLocalizedValue_Windows()
         {
             var testRunner = new TestRunner(_applicationPath);
             return testRunner.RunTestAndVerifyResponse(
                 RuntimeFlavor.Clr,
                 RuntimeArchitecture.x64,
-                "http://localhost:5076",
-                "ResourcesAtRootFolder",
-                "fr-FR",
-                "Bonjour from StartupResourcesAtRootFolder Bonjour from Test in root folder Bonjour from Customer in Models folder");
-        }
-
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
-        public Task Localization_ResourcesAtRootFolder_ReturnLocalizedValue_CoreCLR_NonWindows()
```

Comment:
This test was removed entirely because we already have a Windows and an AllOS version.